### DISCUSSION TITLE
Why are the columns the libraries, not the rows?

### DISCUSSION THREAD

**Discussion by benjamin-kirkbride** [adamghill/unsuckjs.com#9] [General]
Typically you see a comparison like where the columns are the categories, and the rows are the items.

**Comment by valbaca** [comment]
Agreed, especially as the # of items has gone way past the right edge of the page

**Comment by adamghill** [comment]
I'd take a look at a PR to flip the columns and rows. 👍

**Comment by adamghill** [comment]
This has been changed in #24.
